Tests/Functional/Command/CreateIndexCommandTest.php

```diff
@@ -35,7 +42,7 @@ public function getTestExecuteData()
             [
                 'default',
                 [
-                    'timestamp' => false,
+                    'timestamp' => true,
```

Comment:
It doesn't look like additional test when you change current data provider. Please add as real additional statement.